sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -54,17 +54,20 @@
 _NANOSECOND_TO_MICROSECOND = 1000
 _SECOND_TO_MICROSECOND = 1_000_000
 
-T = TypeVar('T')
+ModelT = TypeVar('ModelT')
```

Comment:
I another PR Heejong is using beam.type_hints.TypeVariable.

Any reason to favor one over the other?

Reference PR: https://github.com/apache/beam/pull/17773